### ISSUE TITLE
Balancing Races

### ISSUE THREAD

**Issue by temuchin-** [ospaceteam/outerspace#280]
TL;DR  

- Move asteroid mining tech from humans to cyborgs.

- limit armor to 4+ hull size *4 (this gives a big advantage to humans and a lesser advantage to cyborgs, also helps make large ships more relevant.

- change hull regen from 2%, to 2% small (0); 4% med (1); 8% large (2).  This is an attempt to make bionics useful in medium and large hulls, it won't be enough, but small steps to start.

I have run thirty plus scenarios. Humans are uber powerful, with bionics behind them. Somewheres in distant 3rd is cyborgs, (and are viable only in limited situations).  I wish to be proven wrong, but to do so would require players who are top notch at playing each species. But until then I push forward the idea that for a first step we move asteroid mining from humans to cyborgs. That tech feels like a slap in the face cyborgs. Humans get the best research faciltiies and the best industry options and can pretty much make any system a stronghold they are never lacking in the tools to win.... period. So on top of that to give them the ability to also raise the minerals of planets to 200 is pretty much a slap in the face of cyborgs.

**Comment by dahaic** [comment]
I don't necessarily agree with the latter two points, but moving the asteroid mining to cyborgs sounds interesting.

**Comment by temuchin-** [comment]
Asteroid mining for cyborgs would definitely be a flavor win, and also power them up a bit, but also force them to make difficult choices. Like does the player invest more research in a tech tree that thrives on being very sparse at the higher levels if the player is savvy enough to skip superfluous tech researh. (though there might be arguments made about which techs are superfluous for cyborg. (my quick list is)
Skip battletag and psionic beacon, increase orbit radius, if a player is pinched on research they can easily skip supernova pod, augmented engines slug drive, and devastator drive. Also nano blaster and twin nano turrets are optional. Also only need one level of strip mine. Also ignore chaos cannon, and the larger cyborg beam. And 

After delving through IFleets.py I withdraw my assessment on which species would benefit from armor restrictions, It now seems that cyborgs would get the biggest jump, w/ their brutally efficient ships. I was going off of RoF is king, and instead Damage per Shot reigns supreme. Which definitely favors cyborgs.
